### PR TITLE
Fix for issue #6111

### PR DESCRIPTION
**Pull request by nalakas**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
 import org.wso2.carbon.apimgt.impl.dao.ApiMgtDAO;
 import org.wso2.carbon.apimgt.impl.monetization.DefaultMonetizationImpl;
 import org.wso2.carbon.apimgt.impl.utils.APIUtil;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
```
Comment (chamindias) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java]: Is this used somewhere (seems this is like unused)?

Comment (nalakas) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java]: fixed

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java [hunk 2]
```diff
@@ -81,10 +84,31 @@ public void deleteLabel(String labelId) throws APIManagementException{
...
 
+    /**
+     *
+     * @param label content to check
+     * @return whether label is already added or not
+     * @throws APIManagementException
+     */
+    public boolean isLableNameExists(String tenantDomain, Label label) throws APIManagementException {
+        List<Label> ExistingLables = apiMgtDAO.getAllLabels(tenantDomain);
+        if (!ExistingLables.isEmpty()) {
```
Comment (ruks) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java]: checking list size is redundant here.  Loop continues only if it is not empty.

Comment (nalakas) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java]: fixed in a079997

**Pull request merged**